### PR TITLE
Updated based on PDQm & MHD model

### PR DESCRIPTION
**Pull request by lynnfel**

### CODE REVIEW

File: input/pagecontent/5_testplan.md [hunk 1]
```diff
@@ -1,8 +1,69 @@
...
 
-For FHIR Evaluation use [EVSClient](https://gazelle.ihe.net/content/evsfhirvalidation) or [HL7 FHIR Validator](https://confluence.hl7.org/display/FHIR/Using+the+FHIR+Validator) using the npm package [IHE.ITI.PIXm](package.tgz). 
+This Test Plan page is a prototype.   We expect the maturity of the content will improve over time.  For now, we summarize high level testing scope and available tools. Comments are welcome.
+</div>
 
-For PIXm Consumers wanting to query a [PIXm Manager simulator in Gazelle Patient Manager](https://gazelle.ihe.net/content/pmpixquery-patientidentityconsumer)
+## Introduction
 
-For PIXm Managers wanting to use a [PIXm Consumer simulator in Gazelle Patient Manager]( https://gazelle.ihe.net/content/pmpixquery-pixmanager)
+PIXm specifies a query transaction between two actors.  The transaction between actors specify semantics of the data exchanged.  The PDQm test plan focuses on these semantics and on the expected actions on the server-side actor.
```
Comment (oliveregger) [inline input/pagecontent/5_testplan.md:8]: Instead of 'The PDQm test plan' the PIXm test plan?

Comment (oliveregger) [reply input/pagecontent/5_testplan.md:8]: should we mention also the feed transaction? PIXm specifies a query and a feed transaction between to actors.

File: input/pagecontent/5_testplan.md [hunk 2]
```diff
@@ -1,8 +1,69 @@
...
+  * PIX Consumer, PIX Manager for ITI-83
+  * (future) PIX Source, PIX Manager for ITI-104
+
+#### Gazelle External Validation Service (aka "EVS Client") - Validator
+
+* Provider:  INRIA (Rennes, France), KEREVAL (https://www.kereval.com/), and Mallinckrodt Institute of Radiology (Saint Louis, USA) 
+* Tool location: https://gazelle.ihe.net/EVSClient/home.seam
+* Documentation: https://gazelle.ihe.net/content/evsfhirvalidation
+* Scope of testing: validation using StructureDefinitions for PIXm
+* Other notes: StructureDefinitions available in EVSClient validation are published by IHE here: https://github.com/IHE/fhir/tree/master/StructureDefinition
```
Comment (oliveregger) [inline input/pagecontent/5_testplan.md:56]: I would suggest to remove the line 'StructureDefinitions available in EVSClient', now the StructureDefintion are available in this  implementation guide and I would propose to remove the old ones from https://github.com/IHE/fhir/tree/master/StructureDefinition and let Kereval handle the integration of this ig in EVSClient

Comment (lynnfel) [reply input/pagecontent/5_testplan.md:56]: I thought about this, too.

We should discuss this, I think.    Right now, it's easy for me to tell the kereval people how to configure EVSClient -- just pull all the StructureDefinitions from the single place in gihub under IHE/fhir.

When we move to IGs, do I have to say, "get the MHD StructureDefinitions here, and the PIXm StructureDefinitions there, etc, and then get all of the rest of them from github under IHE/fhir????

Comment (JohnMoehrke) [reply input/pagecontent/5_testplan.md:56]: agree

File: input/pagecontent/5_testplan.md (hunk 1 above)
Comment (lynnfel) [reply input/pagecontent/5_testplan.md:8]: yes PIXm, not PDQm (copy/paste error)

yes, I missed adding the feed.   I can fix both of these.

File: input/pagecontent/5_testplan.md (hunk 2 above)
Comment (oliveregger) [reply input/pagecontent/5_testplan.md:58]: add link to npm package.

**Pull request merged**